### PR TITLE
Add new /sign UI

### PR DESCRIPTION
**Pull request by Patrick1904**
This PR adds an updated UI on the `/sign` route used for signing DApp transactions.

For testing, use the PR staging link:
https://deploy-preview-2215--near-wallet-staging.netlify.app/

Example DApps for testing TX signing:
1. https://app.ref.finance/
2. https://berryclub.io/

### CODE REVIEW

File: packages/frontend/src/components/GlobalStyle.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default createGlobalStyle`
     }
 
     body, p, h1, h2, h3, h4, h5, h6, button, .button, input, select, textarea {
-        font-family: "Inter", Lato, "Lucida Grande", Tahoma, Sans-Serif !important;
+        font-family: "Inter", Lato, "Lucida Grande", Tahoma, Sans-Serif;
```
Comment (MaximusHaximus) [inline packages/frontend/src/components/GlobalStyle.js:60]: Another one bites the dust!

File: packages/frontend/src/components/sign/SignTransferRetry.js [hunk 2]
```diff
@@ -96,20 +94,20 @@ const SignTransferRetry = ({ handleRetry, handleCancel, gasLimit }) => (
         <div className='text'>
             <Translate id='sign.retry.text' />
             <br/><br/>
-            <a href='https://'>
+            <a href='https://docs.near.org/docs/concepts/gas' target='_blank' rel='noreferrer'>
```
Comment (MaximusHaximus) [inline packages/frontend/src/components/sign/SignTransferRetry.js:97]: 💖

File: packages/frontend/src/components/sign/v2/SignTransaction.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+export default ({
+    transferAmount,
+    sender,
+    estimatedFees,
+    availableBalance
+}) => {
+    const isTransferTransaction = new BN(transferAmount).gt(new BN(0));
+    return (
+        <StyledContainer className='transfer-amount brs-8 bsw-l'>
```
Comment (MaximusHaximus) [inline packages/frontend/src/components/sign/v2/SignTransaction.js:88]: Weird class here; are these from semantic UI?

File: packages/frontend/src/components/sign/v2/SignTransactionSummary.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    accountLocalStorageAccountId,
+    availableBalance,
+    estimatedFees,
+    onClickCancel,
+    onClickApprove,
+    onClickMoreInformation,
+    accountUrlReferrer,
+    submittingTransaction
+}) => {
+    const InsufficientBalance = availableBalance && transferAmount && new BN(availableBalance).lt(new BN(transferAmount));
```
Comment (MaximusHaximus) [inline packages/frontend/src/components/sign/v2/SignTransactionSummary.js:58]: :nit: Capitalized var is not typical for a boolean value; suggestion maybe `isBalanceInsufficient`?

File: packages/frontend/src/redux/slices/account/index.js [hunk 5]
```diff
@@ -20,6 +20,7 @@ export const selectAccountLedgerKey = createSelector(selectAccountSlice, (accoun
 
 // balance - state
 export const selectBalance = createSelector(selectAccountSlice, (account) => account.balance || {});
+export const selectAvailableBalance = createSelector(selectBalance, (balance) => balance.balanceAvailable);
```
Comment (MaximusHaximus) [inline packages/frontend/src/redux/slices/account/index.js:23]: 💖

File: packages/frontend/src/routes/SignWrapper.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    const dispatch = useDispatch();
+
+    const [showTransactionDetails, setShowTransactionDetails] = useState(false);
+    const [insufficientNetworkFee, setInsufficientNetworkFee] = useState(false);
+
+    const signFeesGasLimitIncludingGasChanges = useSelector(selectSignFeesGasLimitIncludingGasChanges);
+    const signStatus = useSelector(selectSignStatus);
+    const signCallbackUrl = useSelector(selectSignCallbackUrl);
+    const signMeta = useSelector(selectSignMeta);
+    const transactionHashes = useSelector(selectSignTransactionHashes);
```
Comment (MaximusHaximus) [inline packages/frontend/src/routes/SignWrapper.js:30]: <3 I like the consistency of var naming to selector names.  Could you make a post in the dev channel about this and call this out to everyone else so they can start following this pattern also?

File: packages/frontend/src/routes/SignWrapper.js [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+    }
+
+    if (showTransactionDetails) {
+        return (
+            <SignTransactionDetailsWrapper
+                onClickGoBack={() => setShowTransactionDetails(false)}
+                signGasFee={signGasFee}
+            />
+        );
+    } else {
```
Comment (MaximusHaximus) [inline packages/frontend/src/routes/SignWrapper.js:93]: :nit: can remove this else and the associated indentation

**Review by MaximusHaximus — APPROVED** [review]
This is looking awesome!! nice work.  Just a couple of nit-picky things added in-line that are not blockers.  Let's get this landed this sprint! Feel free to merge if you're happy with it!

File: packages/frontend/src/components/sign/v2/SignTransaction.js (hunk 3 above)
Comment (Patrick1904) [reply packages/frontend/src/components/sign/v2/SignTransaction.js:88]: I've just started adding some classes in GlobalStyle for common things like border radius (brs-8) :)

File: packages/frontend/src/components/sign/v2/SignTransactionSummary.js (hunk 4 above)
Comment (Patrick1904) [reply packages/frontend/src/components/sign/v2/SignTransactionSummary.js:58]: Thanks for catching - fixing:)

**Pull request merged**